docs/management_api.yml

```diff
@@ -308,6 +308,11 @@ paths:
       consumes:
         - multipart/form-data
       parameters:
+        - name: size
+          in: formData
+          description: Size of the artifact file.
```

Comment:
BTW. i think it should be `long` instead of integer. The code expects `int64`and according to swagger spec http://swagger.io/specification/#data-types-12 integer is int32